assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationIntroductoryOverlayNotification.js

```diff
@@ -64,18 +79,20 @@ export default function AudienceSegmentationIntroductoryOverlayNotification() {
 	const scrollToWidgetAndDismissNotification = ( event ) => {
 		event.preventDefault();
 
-		const widgetClass =
-			'.googlesitekit-widget-area--mainDashboardTrafficPrimary';
+		const widgetAreaClass =
+			'.googlesitekit-widget-area--mainDashboardTrafficAudienceSegmentation';
 
 		global.scrollTo( {
-			top: getNavigationalScrollTop( widgetClass, breakpoint ),
+			top: getNavigationalScrollTop( widgetAreaClass, breakpoint ),
 			behavior: 'smooth',
 		} );
```

Comment:
In my testing, the notification is disappearing immediately without the page scrolling.

I've found that wrapping the call to `scrollTo()` in a `setTimeout()` with zero delay works:

```js
		setTimeout( () => {
			global.scrollTo( {
				top: getNavigationalScrollTop( widgetAreaClass, breakpoint ),
				behavior: 'smooth',
			} );
		}, 0 );
```